src/lib/1pwd-vault.ts

```diff
@@ -23,7 +23,8 @@ export async function createVaultCapsuleKeyItem(
   value: string,
   address: string,
   email: string,
-  type: AccountType
+  type: AccountType,
+  isBetaEnv: boolean = false
```

Comment:
maybe we can use here env var called NODE_ENV?
So we can have NODE_ENV = DEVELOPMENT
and NODE_ENV = PRODUCTION on Vercel